app/services/follow_service.rb

```diff
@@ -50,7 +50,7 @@ def mark_home_feed_as_partial!
   end
 
   def following_not_possible?
-    @target_account.nil? || @target_account.id == @source_account.id || @target_account.unavailable?
```

Comment:
The `FollowService` could be (and is) called from other places as well, not just the controller being changed here - should probably leave this in?

Reply:
Have restored the method to what it was originally. If there's something I've missed, the specs should catch it 😅